src/app/technical-record-search/test-result.service.spec.ts

```diff
@@ -74,6 +75,18 @@ describe('TestResultService', () => {
     req.flush({ mockObject: 'mock' });
   });
 
+  it('updateTestResults should update', (done) => {
+    const mock = { mockObject: 'mock' };
+    service.updateTestResults('123', {} as VehicleTestResultUpdate).subscribe((res) => {
+      expect(res).toBeDefined();
+      expect(res).toEqual(mock);
+      done();
+    });
+
+    const req = httpMock.expectOne(req => req.url.includes(`/test-results/123`));
+    req.flush(mock);
+  });
+
```

Comment:
I think we also need to test the rest, testStation, preparers, updateTestResults, etc..

Reply:
As agreed with the team, the code in place was prior to ADR and a technical task has been added to trim the effects where possible
Reference: https://jira.dvsacloud.uk/browse/CVSB-15100